code/game/objects/items/blackout.dm

```diff
@@ -0,0 +1,84 @@
+// Blackout tool, used to trigger massive electricity outttage on ship or station, including connected levels.
+// It may have additional shots to use, but currently balanced to one shot.
+
+/datum/uplink_item/item/tools/blackout
+	name = "High Pulse Electricity Outage Tool"
+	item_cost = 24
+	path = /obj/item/blackout
+	desc = "A device wich can create power virus in terminal, spread it in power network and temporally creating blackout."
+
+/obj/item/blackout
+	name = "high pulse electricity outage tool"
+	desc = "A unknown device, probably only experienced electrics know what this can do."
+	icon = 'icons/obj/items/blackout.dmi'
+	icon_state = "device_blackout-off"
+
+	var/severity = 2
+	var/shots = 1
+	var/lastUse = 0
+	var/Cooldown = (20 MINUTES)
+
+/obj/item/blackout/afterattack(obj/target, mob/user, proximity)
+	if(!proximity)
+		return
+
+	if(!istype(target))
+		return
+
+	target.add_fingerprint(user)
```

Comment:
Simply `add_fingerprint(user)` (that is, adding fingerprints to the blackout tool) seems slightly more reasonable, since you're poking an object with another object (rather than your hands)